### PR TITLE
feat: Generate context json files

### PR DESCRIPTION
**Pull request by WilcoFiers**

### CODE REVIEW

File: plugins/jsonld-contexts/gatsby-node.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const path = require('path')
+var fs = require('fs')
```
Comment (stephenmathieson) [inline plugins/jsonld-contexts/gatsby-node.js]: Why is this a `var`?

File: plugins/jsonld-contexts/gatsby-node.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const webv1 = require(path.resolve(
+  'src',
+  'context',
+  'web-v1.js'
+))
+const contextFiles = ['web-v1', 'ios-v1', 'android-v1']
+
+exports.createPages = async () => {
+  var contextDir = path.resolve('public', 'context')
```
Comment (stephenmathieson) [inline plugins/jsonld-contexts/gatsby-node.js]: `var`?

File: plugins/jsonld-contexts/package.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (stephenmathieson) [inline plugins/jsonld-contexts/package.json]: What is this here for?

File: src/context/android-v1.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+const common = require('./shared')
+const { getPlatformClasses } = require('../schema')
+
+const androidMapping = getPlatformClasses('android').reduce(
+  (androidMapping, { id: className }) => {
```
Comment (stephenmathieson) [inline src/context/android-v1.js]: Why are you calling `id` a `className`? Please add a comment explaining what's happening here.

**Comment by stephenmathieson** [comment]
Also see https://github.com/dequelabs/axrl/pull/31#issuecomment-509231412

File: plugins/jsonld-contexts/package.json (hunk 3 above)
Comment (WilcoFiers) [reply plugins/jsonld-contexts/package.json]: Gatsby plugins require a package.json file. For local plugins they recommend just having an empty object in it. It's kinda weird.

Comment (stephenmathieson) [reply plugins/jsonld-contexts/package.json]: What? That's super weird.

**Pull request closed without merging**